infrastructure-playbooks/rolling_update.yml

```diff
@@ -76,15 +79,11 @@
   become: True
 
   pre_tasks:
-    - include_vars: roles/ceph-common/defaults/main.yml
```

Comment:
These are just redundant because we're using roles. Defining the roles will automatically include these default files.